packages/code-du-travail-frontend/pages/fiche-ministere-travail/[slug].js

```diff
@@ -78,3 +89,12 @@ const Intro = styled(Html)`
     margin-bottom: 0;
   }
 `;
+
+const TabContent = styled(Html)`
```

Comment:
should this be in @socialgouv/react-fiche-service-public ?

Reply:
En fait le component `Html` rajoute une div et ça annule le selecteur d'ou le rajout ici, parce qu'il y est déjà dans UI